app/src/main/java/com/quran/labs/androidquran/service/AudioService.java

```diff
@@ -445,7 +445,7 @@ private void handleIntent(Intent intent) {
   private class ReadGaplessDataTask extends AsyncTask<Integer, Void, SparseIntArray> {
 
     private int mSura = 0;
-    private String mDatabasePath = null;
+    private final String mDatabasePath;
```

Comment:
can we rename to databasePath while you are here?

Reply:
yes we can, changed opened [here](https://github.com/quran/quran_android/pull/1674)